### DISCUSSION TITLE
Editor: can't import audio on Android

### DISCUSSION THREAD

**Discussion by szymsun** [ppy/osu#19966] [Q&A]
### Type

Other

### Bug description

Hi,

I was trying to import an audio to editor on Android and Lazer didn't showed any. Please fix it.

Cheers, Simon. 

### Screenshots or videos

https://user-images.githubusercontent.com/72306436/186677423-49020e70-d03c-436e-8d3b-863395858448.mp4

### Version

2022.821.0

### Logs

[database.log](https://github.com/ppy/osu/files/9425056/database.log)
[network.log](https://github.com/ppy/osu/files/9425057/network.log)
[performance.log](https://github.com/ppy/osu/files/9425058/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9425059/runtime.log)

**Comment by frenzibyte** [comment]
https://github.com/ppy/osu/issues/11911 is probably related but hard to tell what's going on since half of the breadcrumbs of the file selector is cut.

**Reply by szymsun** [reply]
So basically I opened "Downloads" in selection menu and nothing appears. Meanwhile there are 8 sound files in this folder

**Reply by sameerchy1** [reply]
What kind of files tell me the end of the text 
For example
hello.ogg

Just tell me the text after the . Dot

**Reply by szymsun** [reply]
i was trying to import the menu1.ogg file.
After it failed, I've done another attempt with .mp3.

**Reply by szymsun** [reply]
> i was trying to import the menu1.ogg file.
> After it failed, I've done another attempt with .mp3.

Not showed on recording

**Comment by nagi-desuuu** [comment]
What version of android are you using?

**Reply by szymsun** [reply]
Android 11 RKQ1.211001.001
